### ISSUE TITLE
Support for PHPCS ^4.0

### ISSUE THREAD

**Issue by KminekMatej** [doctrine/coding-standard#263]
Currently installing doctrine/coding-standard is not possible if we are using on root project squizlabs/php_codesniffer: 4.0.x-dev

**Comment by greg0ire** [comment]
I don't think we should add support for unstable versions of our dependencies. Why go through the CI complications that will induce when we could wait for a stable version?

**Comment by KminekMatej** [comment]
@greg0ire You're right, somehow I missed the dev appendinx, my bad. Closing

**Comment by pieterocp** [comment]
Given the release of 4.0.0 - https://github.com/PHPCSStandards/PHP_CodeSniffer/releases/tag/4.0.0 

One for a re-opening? (or a new one, but a new issue feels a bit silly imo)

**Comment by simPod** [comment]
This is IMO done, github did not close it for some reason.

**Comment by stof** [comment]
@simPod Github did not close it because the merge closing this has not reached the default branch yet (which is when Github closes issues)

**Comment by simPod** [comment]
Thanks for explaining!

**Comment by stof** [comment]
@simPod speaking of that, do you have any ETA for releasing this support for PHPCS 4.0 ?

**Comment by simPod** [comment]
I'm all in for release but that's up to Doctrine, isn't it?

**Comment by greg0ire** [comment]
I can publish a release tonight. Ping me if I don't.
